### PR TITLE
139/Integrate Okta authentication for vue app.

### PR DESCRIPTION
**Pull request by Ritu-Innostax**
Resolves: #139
Here is the loom:
https://www.loom.com/share/9e25dcfc1c6640faab70ac68aee27cd4

### CODE REVIEW

**Review by manjeetbeniwalinnostax — APPROVED** [review]

**Comment by manjeetbeniwalinnostax** [comment]
Will resolve comments on followup PRs

File: src/envTemplates/.oktaEnv [hunk 1]
```diff
@@ -1,4 +1,10 @@
+<% if(frontEndChoice === "react"){ %>
 REACT_APP_API_URL=Backend API URL
 REACT_APP_OKTA_ISSUER=https://{yourOktaDomain}/oauth2/default
 REACT_APP_OKTA_CLIENT_ID={clientId}
+<% } %>
 
+<% if(frontEndChoice === "vue"){ %>
+VUE_APP_OKTA_CLIENT_ID=your auth0 client Id
+VUE_APP_OKTA_ISSUER=your Auth0 Domain
+<% } %>
```
Comment (Chaman-Raghav) [inline src/envTemplates/.oktaEnv:10]: let's fix, no new line issues in it & all going forward PRs

File: src/index.js [hunk 2]
```diff
@@ -515,22 +500,29 @@ inquirer.prompt(questionnaire).then(async (answers) => {
...
 
     OKTA_FILES_PATHS.forEach((each) => {
-      filePaths = [...filePaths, {
-        source: `${currentPath}/${each.srcFolder}/${each.srcFileName}`,
-        destination: `${frontEnd.path}/${each.destFileName}`
-      }]
+      // filePaths = [...filePaths, {
+      //   source: `${currentPath}/${each.srcFolder}/${each.srcFileName}`,
+      //   destination: `${frontEnd.path}/${each.destFileName}`
+      // }]
```
Comment (Chaman-Raghav) [inline src/index.js:521]: lets remove commented code, if not needed

**Review by Chaman-Raghav — COMMENTED** [review]
approved with minor suggestions

**Pull request merged**